src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -207,63 +207,127 @@ public Task<Either<ActionResult, DeleteReleasePlanViewModel>> GetDeleteReleaseVe
                 });
         }
 
-        public Task<Either<ActionResult, Unit>> DeleteReleaseVersion(Guid releaseVersionId)
+        public Task<Either<ActionResult, Unit>> DeleteReleaseVersion(
+            Guid releaseVersionId,
+            CancellationToken cancellationToken = default)
         {
             return _persistenceHelper
                 .CheckEntityExists<ReleaseVersion>(releaseVersionId)
-                .OnSuccess(_userService.CheckCanDeleteReleaseVersion) // only allows unapproved amendments to be removed
+                .OnSuccess(_userService.CheckCanDeleteReleaseVersion)
                 .OnSuccessDo(async () => await _processorClient.BulkDeleteDataSetVersions(releaseVersionId))
```

Comment:
I checked this with @ntsim the other day, who implied it shouldn't need updating.

Reply:
I think this should work as-is, but worth just clarifying with Jack just in case anyway

Reply:
You need to make any changes. The bulk delete endpoint doesn't assume it is an amendment release. It will go ahead and bulk delete any draft API data sets, and unlink them from the release versions, for any release you pass to the endpoint.